apps/nextjs-website/src/components/templates/SolutionTemplate/SolutionTemplate.tsx

```diff
@@ -29,6 +29,7 @@ export type SolutionTemplateProps = {
     content: BlocksContent;
     products: Pick<Product, 'shortName' | 'slug'>[];
   }[];
+  statsSource?: string;
```

Comment:
statsSource? means `string | undefined` but the codec is returning `string | null`, in the end nothing changes because its converted in js and undefined is as falsey as null, but that would not be the same type